src/pane.coffee

```diff
@@ -577,13 +577,26 @@ class Pane extends Model
     else
       return true
 
+    saveError = (error) =>
+      if error
+        chosen = @applicationDelegate.confirm
+          message: @getMessageForErrorCode(error)
```

Comment:
The other version starting on line 593 definitely identifies the file that has changes and will be saved. This message doesn't seem to do so. Can that be added?